### ISSUE TITLE
Error conversion to int array

### ISSUE THREAD

**Issue by Axou89** [Eclalang/Ecla#81]
There is an error when calling a function which returns an array of int.
For example when using the function DecodeGob(intArr []int) {} from the encoding library, we have :
```ecla
import "encoding";
import "console";

function testDecodeGob() {
    var intArr []int = [71, 111, 98, 32, 105, 115, 32, 97, 119, 101, 115, 111, 109, 101, 33];
    var str string = encoding.decodeGob(intArr);
    console.println(str);
}
 testDecodeGob();
```

It gives the error :
```
panic: interface conversion: interface {} is []interface {}, not []int
```

**Comment by Axou89** [comment]
Lib encoding test :
```ecla
function testEncodeBase64() {
    var intArr []int = [65, 66, 67];
    var str string = encoding.encodeBase64(intArr);
    console.println(str);
}

return the error : panic: interface conversion: interface {} is []interface {}, not []int
```

**Comment by Axou89** [comment]
Lib encoding test :

```ecla
function testEncodeHex() {
    var intArr []int = [65, 66, 67];
    var str string = encoding.encodeHex(intArr);
    console.println(str);
}

return the error : panic: interface conversion: interface {} is []interface {}, not []int
```

**Comment by mkarten** [comment]
the bug was fixed in #109
